spec/models/trainee_spec.rb

```diff
@@ -150,4 +150,31 @@
       it_behaves_like "a working search"
     end
   end
+
+  describe "#ordered_by_date" do
+    let(:trainee_one) { create(:trainee, updated_at: 1.hour.ago) }
+    let(:trainee_two) { create(:trainee, updated_at: 1.day.ago) }
+
+    it "orders the trainess by updated_at in descending order" do
+      expect(Trainee.ordered_by_date).to match_array([trainee_two, trainee_one])
+    end
+  end
+
+  context "trainee status" do
+    let(:trainee_one) { create(:trainee, state: "draft") }
```

Comment:
I think we have factory traits for trainee states, so you can use `create(:trainee, :draft)`, `create(:trainee, : submitted_for_trn)`etc

Reply:
Updated